src/main/java/de/retest/recheck/RecheckProperties.java

```diff
@@ -35,57 +35,57 @@ static RecheckProperties getInstance() {
 	 * Various constants.
 	 */
 
-	static final String PROPERTY_VALUE_SEPARATOR = ";";
-	static final String ZIP_FOLDER_SEPARATOR = "/";
+	String PROPERTY_VALUE_SEPARATOR = ";";
+	String ZIP_FOLDER_SEPARATOR = "/";
 
-	static final String SCREENSHOT_FOLDER_NAME = "screenshot";
-	static final String RECHECK_FOLDER_NAME = "recheck";
-	static final String DEFAULT_XML_FILE_NAME = "retest.xml";
+	String SCREENSHOT_FOLDER_NAME = "screenshot";
+	String RECHECK_FOLDER_NAME = "recheck";
+	String DEFAULT_XML_FILE_NAME = "retest.xml";
 
-	static final String RETEST_FOLDER_NAME = ".retest";
-	static final String RETEST_PROPERTIES_FILE_NAME = "retest.properties";
+	String RETEST_FOLDER_NAME = ".retest";
+	String RETEST_PROPERTIES_FILE_NAME = "retest.properties";
 
-	static final String GOLDEN_MASTER_FILE_EXTENSION = ".recheck";
-	static final String TEST_REPORT_FILE_EXTENSION = ".report";
-	static final String AGGREGATED_TEST_REPORT_FILE_NAME = "tests" + TEST_REPORT_FILE_EXTENSION;
+	String GOLDEN_MASTER_FILE_EXTENSION = ".recheck";
+	String TEST_REPORT_FILE_EXTENSION = ".report";
+	String AGGREGATED_TEST_REPORT_FILE_NAME = "tests" + TEST_REPORT_FILE_EXTENSION;
```

Comment:
Why are these (and the following) not final anymore?

Reply:
Interface constants do not need these modifiers as they are already static final internally (`RecheckProperties.RECHECK_FOLDER_NAME = "bar"` will throw a compile error). I mostly followed the [Intellij Inspection "Unecessary interface modifier"](https://www.jetbrains.com/help/idea/list-of-java-inspections.html).